ebos/ecloutputblackoilmodule.hh

```diff
@@ -1045,6 +1083,24 @@ public:
         if (sSol_.size() > 0)
             sol.insert ("SSOLVENT", Opm::UnitSystem::measure::identity, std::move(sSol_), Opm::data::TargetType::RESTART_SOLUTION);
 
+        if (extboX_.size() > 0)
+            sol.insert ("SS_X", Opm::UnitSystem::measure::identity, std::move(extboX_), Opm::data::TargetType::RESTART_AUXILIARY);
+
```

Comment:
Opm::data::TargetType::RESTART_AUXILIARY would require the user to set the flag --enable-opm-rst-file=true
I don't know the policy of the RESTART_AUXILIARY and RESTART_SOLUTION but maybe it is better to use the RESTART_SOLUTION here since the extended bo model is specific to opm flow.

Reply:
Ok